book/src/themes.md

```diff
@@ -141,6 +141,20 @@ rainbow = ["#ff0000", "#ffa500", "#fff000", { fg = "#00ff00", modifiers = ["bold
 
 Colors from the palette and modifiers may be used.
 
+The `rainbow-brackets` and `rainbow-indents` keys are used for rainbow highlights for matching brackets and indentation guides, respectively.
```

Comment:
Helix automatically fall back it theme keys are of the format `"outer.middle.inner" = ...` (dot-separated), you should use that instead :)